plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -493,6 +504,9 @@ else if (partitionKeys.containsKey(column.getId())) {
                 else if (column.isPathColumn()) {
                     columnAdaptations.add(ColumnAdaptation.constantColumn(nativeValueToBlock(FILE_PATH.getType(), utf8Slice(path.toString()))));
                 }
+                else if (column.isFileModifiedTimeColumn()) {
+                    columnAdaptations.add(ColumnAdaptation.constantColumn(nativeValueToBlock(FILE_MODIFIED_TIME.getType(), packDateTimeWithZone(fileModifiedTime.orElseThrow(), DateTimeZone.getDefault().getID()))));
```

Comment:
Should the zone be configurable or should it be UTC? I think in Delta we do UTC

Reply:
While it's exactly UTC in Delta, Hive connector uses `DateTimeZone.getDefault()`. 

@findepi Do you have any opinion?

Reply:
Thanks for sharing the context. Changed to UTC.